gulpfile.js

```diff
@@ -0,0 +1,103 @@
+var gulp = require('gulp');
+var path = require('path');
```

Comment:
We don't have a formal standard for these files, but I suggest alphabetizing all calls to require() at the top of the file, similar to what we do in the Python files.